support-frontend/app/views/testUsers.scala.html

```diff
@@ -22,9 +22,7 @@ <h1>Test User generator.</h1>
             All other fields can be populated as you wish.
             <h2>All recurring product purchases</h2>
             <ul>
-                <li><strong>First Name:</strong> @key</li>
-                <br><strong>(Optional) Recommended Email (if you do not want to receive emails):</strong> @{key}@@thegulocal.com<br>
```

Comment:
if you want to access manage, you would need to set a password, which means getting the validation email.  So I've removed the option not to get email.